x-pack/plugins/cases/public/components/cases_context/cases_context_reducer.ts

```diff
@@ -44,7 +60,13 @@ export const casesContextReducer: React.Reducer<CasesContextState, CasesContextS
     case CasesContextStoreActionsList.CLOSE_CREATE_CASE_FLYOUT: {
       return { ...state, createCaseFlyout: { isFlyoutOpen: false } };
     }
+    case CasesContextStoreActionsList.OPEN_ADD_TO_CASE_MODAL: {
+      return { ...state, selectCaseModal: { isModalOpen: true, props: action.payload } };
+    }
+    case CasesContextStoreActionsList.CLOSE_ADD_TO_CASE_MODAL: {
+      return { ...state, selectCaseModal: { isModalOpen: false } };
+    }
     default:
-      return state;
+      assertNever(action);
```

Comment:
Do we want to throw an error here? I know it should never get there.

Reply:
This is a typescript-only error. not a runtime error. This guarantees that we don't create actions and forget to put them on the reducer.

Reply:
Still, this gonna throw an error at runtime (check the implementation of `assertNever`). Reducers must be pure and I think throwing an error in reducers could potentially create problems.

Reply:
This error will never throw an error in runtime because this line of code will never be reached.

Reply:
We discuss it offline and we agreed that because it would never reach that part of the code it is ok to have it like this.